src/types/user.ts

```diff
@@ -0,0 +1,6 @@
+export type User = undefined | {
+    email: string,
+    id: string,
+    image: string,
+    name: string,
```

Comment:
Since this file re-defines the `DefaultSession` type we already get from NextAuth library, we should delete this